semantic_conventions/trace/tls.yaml

```diff
@@ -0,0 +1,204 @@
+groups:
+  - id: tls
+    prefix: tls
+    type: span
+    brief: "This document defines semantic conventions for TLS/SSL client and server Spans."
+    attributes:
+      - id: version
+        requirement_level: required
+        brief: >
+          Numeric part of the version parsed from the original string of the
+          negotiated [SSL/TLS protocol version](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_version.html#RETURN-VALUES)
+        type: string
+        examples: ["1.2", "3"]
+      - id: version_protocol
+        requirement_level: required
+        brief: >
+          Normalized lowercase protocol name parsed from original string of the
+          negotiated [SSL/TLS protocol version](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_version.html#RETURN-VALUES)
+        type:
+          allow_custom_values: false
+          members:
+            - id: ssl
+              value: ssl
+            - id: tls
+              value: tls
+            - id: unknown
+              value: unknown
```

Comment:
```suggestion
```

if we don't require it and allow custom values, there is no need in unknown value

Reply:
makes sense